src/constants.js

```diff
@@ -267,3 +268,12 @@ export const FEATURE_TOGGLE_DEFAULTS = {
   [WALLET_SERVICE_FEATURE_TOGGLE]: false,
   [ATOMIC_SWAP_SERVICE_FEATURE_TOGGLE]: false,
 };
+
+/**
+ * This property filters the full exported scss properties and exposes only the ones
```

Comment:
Is there any downside to exposing all exported variables from css? I'm just wondering about this because in case we decide to add a new variable there, we would need to change it here as well. If we do a direct export, we must change one place only.

Reply:
In the current webpack loading behavior all elements from the `scss` file, both class definitions and colors in a garbled notation, are exposed to the javascript environment together with the ones we explicitly export through `:export {}`.

So, the downside of leaving this unfiltered is making the `constants.colors` property have hundreds of elements inside it that are not colors, which can confuse developers that try to read all constants values. 

Other than that, removing the filter will have no other downsides on the application.